src/extension.ts

```diff
@@ -1145,9 +1198,13 @@ function execKubernetes() {
 
 async function terminalKubernetes(explorerNode?: explorer.ResourceNode) {
     if (explorerNode) {
```

Comment:
invert the condition so we can return early w/ `execKubernetesCore`

Reply:
Disagree: although the 'not an explorer node' branch is shorter, it's not a true early exit scenario and the two branches have equal weight, so I prefer the positive if condition.